lit/docs/install/concourse-cli.lit

```diff
@@ -9,11 +9,20 @@ contains the following files:
 
 \codeblock{sh}{{
 concourse/bin/concourse
-concourse/bin/gdn            # on Linux
+concourse/bin/gdn            # Linux only
 concourse/fly-assets/...
-concourse/resource-types/... # on Linux
+concourse/resource-types/... # Linux only
 }}
 
+The Linux release is the largest among all the platforms because it contains
+some of the most commonly used resource types like the
+\link{git}{https://github.com/concourse/git-resource},
+\link{time}{https://github.com/concourse/time-resource/}, and
+\link{registry-image}{https://github.com/concourse/registry-image-resource/}
+resources. Resources only run on Linux workers, that's why the other platforms
+are not bundled with resources; resources don't currently exist for non-linux
+platforms.
+
 When extracted, the \code{concourse} binary will auto-discover its sibling
```

Comment:
Can you also detail where these resources are located and how to figure out what resource versions are deployed on each worker? 

- `fly workers --details` which gives resource types but no versions
- `fly curl api/v1/workers` which gives bulk response for resource types + versions for every worker
- hopping onto a worker and reading the `metadata.json`

Reply:
Great idea. I'll do this in the worker section.